packages/ui-causal-graph-editor/src/shared/use-causal-graph-editor.tsx

```diff
@@ -108,7 +141,7 @@ export default function useCausalGraphEditor(
 
         lastParentData.current = graphData;
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [graphData]);
+    }, []);
```

Comment:
Why are we removing the dependency here? Without it it will always only compare to the original graph data

Reply:
oops it should not have been, added it back